lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -254,10 +279,11 @@ def test_missing_keys(self, client, server_config, pbench_token):
         """
 
         def missing_key_helper(keys):
-            response = client.post(
+            response = self.make_request_call(
```

Comment:
Is this method defined inline only used for `POST` operations?  It appears so.  We should only run this test for `POST` operations.

Reply:
Yeah; at least, it's more complicated for `GET` since "missing parameters" are URI or query parameters rather than JSON body. That's not currently handled. For example, if there's a required dataset ID in the URL, we could drive a test case without it (e.g., `GET /datasets/delete` where the code requires `/datasets/delete/<resource_id>`). Or maybe that test should just be special cased (which I think it is), and we just skip this test case for `GET` APIs.